pipeline/indirect/indirect.py

```diff
@@ -2,6 +2,8 @@
 import pandas as pd
 import pycountry
 from climada_petals.engine import SupplyChain
+from exposures.utils import root_dir
+project_root = root_dir()
```

Comment:
No reason to put this here I guess, just use it in your function.

Reply:
yes, moved it to the function